static/templates/settings/account-settings.handlebars

```diff
@@ -212,6 +212,17 @@
             <div id="user_api_key_error text-error">
             </div>
         </div>
+        <div class="">
+            <h3>{{t "Export my messages" }}</h3>
+            <div id="user_export_data_message" class="alert alert-info"></div>
+            <form class="grid">
+                <div class="input-group">
+                    <button type="submit" class="button rounded w-200 input-size" id="user_export_data">
+                        {{t 'Export my messages' }}
+                    </button>
+                </div>
+            </form>
+        </div>
```

Comment:
I think we might want to move this into the below section and title it was "Account options."